### PR TITLE
feat(react-native): upgrade Node/NPM and remove watchman, exposed ports and the custom user

### PR DESCRIPTION
**Pull request by GuillaumeAmat**
Hi there,

I did some small things in the React Native image:

* Upgrade Node.js from v7 to v12 (LTS)
* Remove Watchman
* Remove some exposed ports
* Remove the custom user because of issues with root-owned files (eg: cached dependencies from a previous CI job, based on another Docker image)

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -272,9 +272,11 @@ python:
 
 react-native:
   "1":
+    build_args:
```
Comment (mremi) [inline config.yml]: duplicated

File: config.yml [hunk 2]
```diff
@@ -272,9 +272,11 @@ python:
 
 react-native:
   "1":
```
Comment (mremi) [inline config.yml]: should be updated

**Comment by GuillaumeAmat** [comment]
It should be good @mremi

**Comment by GuillaumeAmat** [comment]
@mremi is that ok for you? Thanks

**Comment by mremi** [comment]
Yep, I let you squash the commits and so write a proper commit message

**Comment by GuillaumeAmat** [comment]
👌

**Comment by GuillaumeAmat** [comment]
All good @mremi

**Pull request merged**